### PR TITLE
Langserve template updates

### PR DESCRIPTION
**Pull request by bracesproul**

### CODE REVIEW

File: templates/neo4j-cypher/pyproject.toml
Comment (bracesproul) [inline templates/neo4j-cypher/pyproject.toml:1]: is this export_module correct? `export_module = "neo4j.cypher"`

File: templates/summarize-anthropic/summarize_anthropic.ipynb [hunk 1]
```diff
@@ -127,6 +127,11 @@
     " \n",
     "We can also use remote runnable to call it.\n",
     "\n",
+    "In `server.py`, set -\n",
```
Comment (hwchase17) [inline templates/summarize-anthropic/summarize_anthropic.ipynb:130]: not sure this is the best place to put this, but i think general some of these notebooks need cleaning up

File: templates/stepback-qa-prompting/pyproject.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "stepback_qa_prompting"
+name = "stepback-qa-prompting"
```
Comment (hwchase17) [inline templates/stepback-qa-prompting/pyproject.toml:2]: fyi - this doesnt matter

Comment (hwchase17) [reply templates/stepback-qa-prompting/pyproject.toml:2]: infact, this is actually NOT desirable (the `new` command creates it like this, so we should be consistent)

File: templates/neo4j-cypher/pyproject.toml
Comment (hwchase17) [reply templates/neo4j-cypher/pyproject.toml:1]: shoulds probably be changed

File: templates/anthropic-functions/pyproject.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "anthropic-function"
+name = "anthropic-functions"
```
Comment (hwchase17) [inline templates/anthropic-functions/pyproject.toml:2]: nice

**Comment by hwchase17** [comment]
there are very few things here that are correct, so im going to close this and start a new branch and cherry pick the correct stuff

**Pull request closed without merging**